lib/backends/sdc/containers.js

```diff
@@ -2017,15 +2476,35 @@ function getContainers(opts, callback) {
             });
 
             if (filterErr) {
-                callback(filterErr);
+                cb(filterErr);
                 return;
             }
+
+            stash.objects = objects;
+            cb();
+        }, function _limitResults(stash, cb) {
+            var objects = stash.objects;
+
+            if (opts.limit > 0) {
+                objects = objects.slice(0, opts.limit);
+            }
+
+            // last cb in chain, so we pass objects to results
+            cb(null, objects);
         }
+    ]}, function _pipelineCb(p_err, p_results) {
```

Comment:
Are `p_err` and `p_results` prefixed with `p_` to avoid shadowing other variables named `err` and `results`? If so, is there some refactoring that wouldn't be too time consuming that would allow us to not do that? Otherwise my concern is that it's a missed opportunity to make the code not too confusing.